contributors/guide/owners.md

```diff
@@ -56,6 +61,13 @@ reviewers:
   - alice
   - carol   # this is another comment
   - sig-foo # this is an alias
+security_contacts:
+  - github: alice
+    email: alice@mail.com
+    slack: @alice
+  - github: dan
+    email: null
+    slack: null
 ```
```

Comment:
Are we "allowed" to publish these?  I always here this type of public identification mapping is somehow not allowed by GitHub ToS?

Will all three of {github,email,slack} be required?

Reply:
I'm pretty sure we can publish those -  github/email is mapped already via cncf gitdm from commits, and we have github -> slack mappings for slack user-groups in the slack-config.

Many people publish that info themselves for their personal website as well.

Reply:
> Will all three of {github,email,slack} be required?

No, I updated the descriptions to be clearer that only GitHub username is required. The other fields are optional.

> I'm pretty sure we can publish those - github/email is mapped already via cncf gitdm from commits, and we have github -> slack mappings for slack user-groups in the slack-config.

That was my feeling as well. I also reached out to [GitHub's privacy team](https://support.github.com/contact/privacy) for some input but I'm not sure if this is within their scope. I'll share any reply I get.

Also, I think the intention is for individuals to add update _their own_ email/slack IDs. The bulk updates that add `security_contacts` to OWNERS will have `email` and `slack` set to `null` for all users.

Reply:
Got a reply from github Trust & Safety team:

> Thanks for reaching out. Our Acceptable Use Policies address your question in the following way:
> 
>     While using the Service, under no circumstances will you:
> 
>     violate the privacy of any third party, such as by posting another person's personal information without consent.
> 
> Additionally, Section 6 may apply, depending on where you're getting the information:
> 
>     If you collect any User Personal Information from the Service, you agree that you will only use that User Personal Information for the purpose for which that User has authorized it.
> 
> If you're gathering and using public information from GitHub and are receiving consent from each of the data subjects, our Privacy Statement includes some further requirements. The following is especially important:
> 
>     We expect you to reasonably secure any User Personal Information you have gathered from GitHub, and to respond promptly to complaints, removal requests, and "do not contact" requests from GitHub or GitHub users.
> 
> I hope this is helpful. Please let us know if we can answer any other questions.

I think the current proposal be okay then. Users would need to add their own contact info, we can't add any on their behalf.

Reply:
One other thing I forgot to mention with Slack, names are not unique in there :|  There can be multiple mrbobbytables for examples, thats why we have the mapping of slack name to underlying slack ID in usergroup management. I would avoid including that at all as you could easily ping the wrong person.

Reply:
Ah, yeah I see what you mean looking in `communication/slack-config`. Some other options might be either to ask for the actual slack ID instead (downside: not easily enforceable), or enhance the spec to have keys for both slack name and slack ID (downside: messier, still not really enforceable).

So I think your suggestion to remove the `slack` field entirely is probably the most sensible. I'll hold back on the change for a day in case there are other ideas/opinions.

Reply:
when this is place I am good to lgtm on this PR.

Reply:
Removed the 'slack` field